x-pack/plugins/lens/public/shared_components/axis/ticks/axis_ticks_settings.test.tsx

```diff
@@ -6,56 +6,41 @@
  */
 
 import React from 'react';
-import { act } from 'react-dom/test-utils';
-import { mountWithIntl as mount } from '@kbn/test-jest-helpers';
-import { EuiSwitch, EuiSwitchEvent } from '@elastic/eui';
 import { AxisTicksSettings, AxisTicksSettingsProps } from './axis_ticks_settings';
-
-jest.mock('lodash', () => {
-  const original = jest.requireActual('lodash');
-
+import { render, screen } from '@testing-library/react';
+
+const renderAxisTicksSettings = (propsOverrides?: Partial<AxisTicksSettingsProps>) => {
+  const rtlRender = render(
+    <AxisTicksSettings
+      isAxisLabelVisible={true}
+      axis="x"
+      updateTicksVisibilityState={jest.fn()}
+      {...propsOverrides}
+    />
+  );
   return {
-    ...original,
-    debounce: (fn: unknown) => fn,
+    tickLabelsSwitch: screen.getByLabelText('Tick labels'),
+    ...rtlRender,
   };
```

Comment:
I like this pattern, but is it too much here? Not necessarily objecting, just asking the question.

Reply:
You mean to not create `tickLabelsSwitch` property? You might be right, it's a very thin line...

Reply:
> You mean to not create tickLabelsSwitch property? You might be right, it's a very thin line...

Yes and also I mean not wrapping the `render` call in `renderAxisTicksSettings` at all.

> You might be right, it's a very thin line...

It is. I guess we'll get a sense for it as these things grow.